### PR TITLE
Fixing error in SQL syntax on /api/index.php/item/get?id=xxx

### PR DESCRIPTION
**Pull request by nilsteampassnet**
Merged suggestion from #4404

### CODE REVIEW

File: includes/config/include.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 define('TP_VERSION', '3.1.2');
 define("UPGRADE_MIN_DATE", "1727110744");
-define('TP_VERSION_MINOR', '134');
+define('TP_VERSION_MINOR', '5');
```
Comment (corentin-soriano) [inline includes/config/include.php:31]: 3.1.2.5 ?

**Comment by corentin-soriano** [comment]
I think there is another issue:
image

**Comment by nilsteampassnet** [comment]
> I think there is another issue

Not sure to understand.

**Comment by corentin-soriano** [comment]
> > I think there is another issue
> 
> Not sure to understand.

I don't see any call to getAction(). I think @ego-kyriarchos, added the missing call too.

**Comment by nilsteampassnet** [comment]
> > > I think there is another issue
> > 
> > 
> > Not sure to understand.
> 
> I don't see any call to getAction(). I think @ego-kyriarchos, added the missing call too.

It is used through `bootstrap.php` file.
You have instruction: `$strMethodName = $actions[0] . 'Action';`
with `$actions[0]` coming from the URL, and in this case the value would be `get`.
So  `$strMethodName = 'getAction';` which is the name of the called function

**Comment by corentin-soriano** [comment]
> > > > I think there is another issue
> > > 
> > > 
> > > Not sure to understand.
> > 
> > 
> > I don't see any call to getAction(). I think @ego-kyriarchos, added the missing call too.
> 
> It is used through `bootstrap.php` file. You have instruction: `$strMethodName = $actions[0] . 'Action';` with `$actions[0]` coming from the URL, and in this case the value would be `get`. So `$strMethodName = 'getAction';` which is the name of the called function

Thank you for the clarification!

**Review by corentin-soriano — APPROVED** [review]
Can you push the TP_MINOR_VERSION before merging?

**Comment by nilsteampassnet** [comment]
The minor version will be fixed directly under master branch as it is a counter based upon the number of commits inside the branch ;)

**Pull request merged**